XiEditor/AppDelegate.swift

```diff
@@ -72,17 +51,28 @@ class AppDelegate: NSObject, NSApplicationDelegate {
             if let obj = params as? [String : AnyObject], let update = obj["update"] as? [String : AnyObject] {
                 guard let tab = obj["tab"] as? String
                     else { print("tab missing from update event"); return }
-                guard let appWindowController = appWindowControllers[tab]
-                    else { print("tab " + tab + " not registered"); return }
-                appWindowController.editView.updateSafe(update: update)
+                
+                //FIXME: a better way to get a document
```

Comment:
Okay, made the change. I think during cleanup I'd like to use some sort of dispatch table mapping "tab names" to documents.

I'm also wondering if "tab name" is the clearest way of describing what a tab name is doing? My understanding is that it's essentially a buffer identifier?